### ISSUE TITLE
Challenge_gov Dependabot alerts

### ISSUE THREAD

**Issue by r-bartlett-gsa** [GSA/Challenge_gov#1475]
- [x] Review [dependabot alerts](https://github.com/GSA/Challenge_gov/security/dependabot) and resolve items that apply to challenge_gov
  - [x] Document below which items are being worked on / resolved
- [x] Update the items that do not apply to challenge_gov

**Comment by jairoanaya** [comment]
@r-bartlett-gsa dependabots have been addressed succesfully.

**Comment by r-bartlett-gsa** [comment]
@jairoanaya Is there any particular testing we need to do for these dependabots, or are these updates ready to be pushed to production?

**Comment by jairoanaya** [comment]
@r-bartlett-gsa, those dependabots are NOT in production, as I mentioned on the retrospective meeting on 01/28/25, and related to #1500 they are being addressed only on dev (which still un-synchronized) - which I understand is blocked by #1430 - please refer to my comments https://gsa-tts.slack.com/archives/C086N4GS1F0/p1738076371363549 

Please refer to the following PR

https://github.com/GSA/Challenge_gov/pull/1473
https://github.com/GSA/Challenge_gov/pull/1487

**Comment by jairoanaya** [comment]
@r-bartlett-gsa these dependabots have been resolved on PR https://github.com/GSA/Challenge_gov/pull/1579

**Comment by jairoanaya** [comment]
@TCKapGrp this has been resolved on https://github.com/GSA/Challenge_gov/pull/1579

**Comment by jairoanaya** [comment]
@jdonis new PR has been created to update all pending dependabots https://github.com/GSA/Challenge_gov/pull/1623

**Comment by afeijoo** [comment]
Archiving repo; no longer applicable.
